README.md

```diff
@@ -135,7 +135,7 @@ using these `20 tokenB` we can get `10 tokenA` from the buy order, earning `2 to
 
 ### Running the SOR using Docker
 
-A ready-to-run, containerized version of the Smart Order Router is available via the [GitHub Container Registry](ghcr.io/geniusyield/smart-order-router:latest).
```

Comment:
On https://github.com/geniusyield/smart-order-router/pkgs/container/smart-order-router there is a link under "Running the SOR using Docker" called GitHub Container Registry.

This is broken. So this must be fixed.